src/components/Navbar/MenuItems.js

```diff
@@ -0,0 +1,33 @@
+export const MenuItems = [
+  {
+    title: "Home",
+    url: "/",
+    className: "nav-links",
+  },
+  {
+    title: "Products",
+    url: "/products",
+    className: "nav-links",
+  },
+  {
+    title: "Contact Us",
+    url: "/contact",
+    className: "nav-links",
+  },
+  {
+    title: "Sign Up",
+    url: "/signup",
```

Comment:
Is the 'Sign Up' page supposed to be for registering a new user?
If so, this url needs to go too '/register'

Reply:
Agreed. I was working on the Togglewithit branch and updated the link in that branch.